core/src/mindustry/world/consumers/ConsumeLiquidViscosity.java

```diff
@@ -0,0 +1,32 @@
+package mindustry.world.consumers;
+
+import mindustry.type.*;
+import mindustry.world.meta.*;
+
+public class ConsumeLiquidViscosity extends ConsumeLiquidFilter{
```

Comment:
Did anyone specifically ask for these? It seems far too specific; I don't see anyone ever needing to care about a liquid's heat capacity or viscosity. At that point, you're better off with some kind of special filter subclass that has specific efficiencies for specific items.

Reply:
idk, I just added them for consistency since others stats have things for this

Reply:
I don't think the 3 extra classes are necessary, it's just extra stuff to maintain.